broker/src/main/java/io/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -89,7 +89,7 @@
   private LogStream logStream;
   private Role raftRole;
   private SnapshotReplication stateReplication;
-  private PersistedSnapshotStore persistedSnapshotStore;
+  private final ActivePersistedSnapshotStore persistedSnapshotStore;
```

Comment:
Why is this called active? I assume because you can actively create a snapshot ? :D

Reply:
We agreed on `ConstructableSnapshotStore`, if someone has a better name feel free to propose.